auth/admin.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.models import User
+
+from auth.models import UserProfile
+
+
+# Define an inline admin descriptor for UserProfile model
+class UserProfileInline(admin.StackedInline):
+    model = UserProfile
+    can_delete = False
+
+
+# Define a new User admin
+class UserAdmin(BaseUserAdmin):
+    inlines = (UserProfileInline, )
+
+# Re-register UserAdmin
```

Comment:
what is the purpose of this?

Reply:
Add `ProfileUser` fields  to `DjangoUser` as an in-line.